cmd/k3s/main.go

```diff
@@ -44,7 +44,7 @@ func main() {
 		cmds.NewCRICTL(externalCLIAction("crictl", dataDir)),
 		cmds.NewCtrCommand(externalCLIAction("ctr", dataDir)),
 		cmds.NewCheckConfigCommand(externalCLIAction("check-config", dataDir)),
-		cmds.NewEtcdSnapshotCommand(etcdsnapshotCommand, cmds.NewEtcdSnapshotSubcommands(etcdsnapshotCommand)),
+		cmds.NewEtcdSnapshotCommand(etcdsnapshotCommand, cmds.NewEtcdSnapshotSubcommands(etcdsnapshotCommand, etcdsnapshotCommand)),
```

Comment:
We're passing the same thing twice?

Reply:
Yes, since in the wrapper they're all just dummy internalCLIActions that result in the actual binary being extracted out and run. We need to pass a valid set of subcommand actions so that they show up in the help text.

The whole command-factory thing is ugly but required to avoid linking all the core code into the rke2 wrapper binary.